test/root/root_api_test.rb

```diff
@@ -34,4 +34,9 @@ def test_version
     get "/version"
     assert_equal({"version" => Proxy::VERSION}, JSON.parse(last_response.body))
   end
+
+  def test_version_header
+    get "/version"
```

Comment:
It would work only with `/version` and `/features`. I guess I could add another test for features to